docs/observatorium/loki/README.md

```diff
@@ -0,0 +1,52 @@
+## Loki Access
```

Comment:
What do you think about adding a quick intro sentence on what Loki is, and a link to docs. We can do this for the other components too. It can be a one liner with a link.

Reply:
I think that's a good idea :+1:

Reply:
added